src/database/DataTypes/Item.js

```diff
@@ -240,11 +240,16 @@ export class Item extends Realm.Object {
   openVialWastage(fromDate) {
     if (!this.isVaccine || !fromDate) return 0;
 
-    return UIDatabase.objects('TransactionBatch')
+    const transactions = UIDatabase.objects('TransactionBatch')
       .filtered("transaction.otherParty.code == 'invad'")
       .filtered("option.type == 'openVialWastage'")
-      .filtered('transaction.confirmDate >= $0', fromDate)
-      .sum('doses');
+      .filtered('transaction.confirmDate >= $0', fromDate);
+
+    if (transactions.length) {
+      return transactions.sum('doses');
+    }
+
+    return 0;
```

Comment:
```suggestion
    return transactions.length ? transactions.sum('doses') : 0;
```

Reply:
Thanks, I'm not sure whether I find it easier or harder 😆 . The shorter form does seem to be used quite prolifically through the app though so maybe I should conform 😸  bb1b00f